resources/views/navbar/items/dropdown-item.blade.php

```diff
@@ -0,0 +1,87 @@
+@props([
+    'label',
+    'description' => null,
+    'route'       => null,
+    'href'        => null,
+    'routeParams' => [],
+    'icon'        => null,
+    'disabled'    => false,
+    'external'    => false,
+    'tooltip'     => null,
+])
+
+@php ($isCurrent = $disabled === false && $route && url()->full() === route($route, $routeParams))
+
+<div class="flex">
+    <div @class([
+        'w-1 -mr-1 z-10',
+        'bg-theme-primary-600' => $isCurrent,
+    ])></div>
+
+    <a
+        @if ($disabled)
+            href=""
+            aria-disabled="true"
+            tabindex="-1"
```

Comment:
@alexbarnsley I made a research on that topic and it would be better to avoid this because it always announced as a Link to screen-readers but it's not interactable (and it's not what we expect).

When an anchor tag is disabled, it loses its behavior becoming a simple string. This is why W3C doesn't added a way to disable an anchor tag.

In our case, the best solution is to use a `<span>` and style it as "disable".

You can find more details in [this article](https://css-tricks.com/how-to-disable-links/#just-dont-do-it).

Reply:
:thinking: that makes sense. Will update it, thanks!

Reply:
@leMaur i used div instead of span unless you have a specific reason for using span? :thinking: